test/integration-test.sh

```diff
@@ -47,7 +47,7 @@ run_command(){
   local wait_time=${4:-0}
   local step_time=${5:-1}
   local end_time_ns=${6:-0}
-  local cmd="curl --silent --max-time 360 -X POST --url ${endpoint}/api/v1/command --data"
+  local cmd="curl --silent --max-time 360 -X POST --header X-Auth-Token:e2e-api-token --url ${endpoint}/api/v1/command --data "
```

Comment:
why is this necessary if we are running these nodes without authentication?

Reply:
They all run with auth to reduce complexity.